blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -174,8 +174,13 @@
     // 3 days - this matches the TTL for our metrics_full table, we don't accept anything older than the TTL.
     BEFORE_CURRENT_COLLECTIONTIME_MS("259200000"),
     // 10 minutes
-    AFTER_CURRENT_COLLECTIONTIME_MS("600000");
+    AFTER_CURRENT_COLLECTIONTIME_MS("600000"),
 
+    // Cross-Origin Resource Sharing
+    CORS_ALLOWED_ORIGINS(""),
```

Comment:
@goru97 do we take the regular expression from the config and translate to it to the response header values somehow to be returned?  If so, can you provide me with an example?  Or would it be that if a regular expression is provided in the config, that same regular expression would be returned in the response header (i.e. for Access-Control-Allow-Origin).